### DISCUSSION TITLE
is there any way to use this extension and term combining?

### DISCUSSION THREAD

**Discussion by gsgoldma** [adieyal/sd-dynamic-prompts#226] [General]
The syntax for combinational generation here and the original use of  {term1 |term2} actually combining the embeddings is mutually exclusive, or is there some workaround I don't know about?

**Comment by adieyal** [comment]
Do you mean [alternating text]( https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features#alternating-words)? That syntax uses square brackets.
